code/modules/religion/religion_sects.dm

```diff
@@ -193,3 +193,148 @@
 	adjust_favor(round(the_cell.charge/3000), L)
 	to_chat(L, "<span class='notice'>You offer [the_cell]'s power to [GLOB.deity], pleasing them.</span>")
 	qdel(I)
+
+
+//Fallen Sect
+
+/datum/religion_sect/fallen
+	name = "Sleeping Gods"
+	desc = "A sect that has fallen into dormancy."
+	convert_opener = "The gods have been silent. Where have you gone?!<br>You have no powers... But, in the darkest hour you will rise to save them all."
+	alignment = ALIGNMENT_GOOD //ancient defenders?
+	altar_icon_state = "sacrificealtar" //just no candles or the like, pretty dull altar
+	/// the current security level as a number, used to give out rewards
+	var/chaos_level = 0
+
+/datum/religion_sect/fallen/on_select()
+	RegisterSignal(SSdcs, COMSIG_GLOB_SECURITYLEVEL_CHANGED, .proc/alert_change)
+
+/datum/religion_sect/fallen/proc/alert_change(new_level)
+	var/increase = chaos_level < new_level ? TRUE : FALSE
+	var/godmessage = ""
+	var/wings = FALSE
+	chaos_level = new_level
+	switch(chaos_level)
+		if(SEC_LEVEL_GREEN)
+			godmessage = "[GLOB.deity] is silent, any murmurs of his power have vanished."
+		if(SEC_LEVEL_BLUE)
+			if(increase)
+				godmessage = "You feel a faint buzz of holy energies for just a moment."
+			else
+				godmessage = "You feel your powers fade as [GLOB.deity] returns to it's slumber..."
+		if(SEC_LEVEL_RED)
+			if(increase)
+				godmessage = "You feel your powers surge as [GLOB.deity] awakens!"
+			else
+				godmessage = "[GLOB.deity] is satisfied with the end of the delta event. The wings are your reward."
+		if(SEC_LEVEL_DELTA)
+			godmessage = "You have been given the full power of [GLOB.deity] to save the station from crisis. [GLOB.deity] watches over you."
+			wings = TRUE
+	if(godmessage)
+		for(var/i in GLOB.player_list)
+			if(!isliving(i))
+				continue
+			var/mob/living/am_i_holy_living = i
+			if(!am_i_holy_living.mind?.holy_role)
+				continue
+			to_chat(am_i_holy_living, "<span class='warning'>[godmessage]</span>")
+			if(wings && ishuman(am_i_holy_living))
+				var/mob/living/carbon/human/holyhumie = am_i_holy_living
+				holyhumie.dna.species.GiveSpeciesFlight()
+
+/datum/religion_sect/fallen/sect_bless(mob/living/L, mob/living/user)
+	switch(chaos_level)
+		if(SEC_LEVEL_GREEN, SEC_LEVEL_BLUE)
+			. = FALSE
+			to_chat(user, "<span class='warning'>[GLOB.deity]... where are you?!</span>")
+			SEND_SIGNAL(L, COMSIG_ADD_MOOD_EVENT, "forsaken", /datum/mood_event/forsaken)
+		if(SEC_LEVEL_RED, SEC_LEVEL_DELTA)
+			if(!ishuman(L))
+				..()
+				return TRUE
+			. = TRUE
+			var/mob/living/carbon/human/H = L
+			for(var/X in H.bodyparts)
+				var/obj/item/bodypart/BP = X
+				if(BP.status == BODYPART_ROBOTIC)
+					to_chat(user, "<span class='warning'>[GLOB.deity] refuses to heal this metallic taint!</span>")
```

Comment:
might be spammy with multiple aug limbs

Reply:
might just cut it OR send it to the actual blessed person

Reply:
why do you even have this its only there because puritans dislike robots, why does fallen do this